frontend/views/containers/modals/InviteByLink.vue

```diff
@@ -42,8 +42,15 @@ export default {
 }
 
 .c-link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
   margin-top: $spacer-lg;
   margin-bottom: $spacer-sm;
+
+  ::v-deep .c-copy-button {
+    margin-left: $size-3/2;
```

Comment:
We don't use `$size-x` variables for spacing, but for text size.
When implementing a design, we with @mmbotelho use a spacing sytem, 4px, 8px, 16px, 24px, 32px, etc.. If you see something that doesn't match that, use the closest value and double-check with @mmbotelho to see if it's the correct one. I believe that in `InviteLinkToCopy` the space between the link and the button is the same anywhere, so there's no need to override it here.

Reply:
Fixed!